src/processors/default/createERC721NFTsFromTransfersProcessor.ts

```diff
@@ -31,43 +32,68 @@ const processorFunction = (contracts: NftFactory[]) =>
         throw 'buildNFTId not specified'
       }
 
+      const contractAddress = formatAddress(contract.id);
```

Comment:
hmm i think we weren't formatting the address before right?

i guess this means that if someone is following a collector today, they are following an unformatted address and when they change to formatted addresses, they may lose their follows?

i'm not sure the formatting/api from the frontend, so haven't doubled check if this will actually happen, but I think it's not a big deal so I don't think we should worry about handling it.

Reply:
Ok, so just to clarify, should we keep as is or revert to leaving it unformatted?

Reply:
keep as is